warehouse/models/mart/gtfs_guidelines/guideline_checks.md

```diff
@@ -24,4 +24,5 @@ Here is a list of currently-implemented checks:
 | Trip updates RT feed is listed on feed aggregator transit.land | Feed Aggregator Availability | Trip updates RT feed is listed on feed aggregator transit.land on the given day. Due to current limitations, this check will be less reliable for agencies with multiple trip updates feeds listed in agencies.yml. If one of the feeds listed is on transit.land, it will pass for all trip updates feeds. |
 | Service alerts RT feed is listed on feed aggregator transit.land | Feed Aggregator Availability | Service alerts RT feed is listed on feed aggregator transit.land on the given day. Due to current limitations, this check will be less reliable for agencies with multiple service alerts feeds listed in agencies.yml. If one of the service alerts feeds listed is on transit.land, it will pass for all feeds. |
 | Every trip in trips.txt has a shape_id listed | Accurate Service Data | Every row in trips.txt has a non-null value in the shape_id column. |
+| Include tts_stop_name entries in stops.txt for stop names that are pronounced incorrectly in most mobile applications. | Accurate Accessibility Data | For every stop_name in stops.txt containing a commonly mispronounced word, there is a non-null tts_stop_name field. |
```

Comment:
Worth mentioning here that it also checks for whether it's identical to `stop_name`?

Reply:
Thanks! I included this in an overhaul of the description. Might as well be very specific.